src/python/espressomd/reaction_ensemble.pyx

```diff
@@ -720,10 +769,14 @@ cdef class WidomInsertion(ReactionAlgorithm):
 
     def measure_excess_chemical_potential(self, reaction_id=0):
         """
-        Measures the excess chemical potential in a homogeneous system.
+        Measures the excess chemical potential in a homogeneous system for the provided reaction_id.
+        Please register the insertion moves first via calling the method add_reaction (with only product types specified).
         Returns the excess chemical potential and the standard error for the
-        excess chemical potential. It assumes that your samples are
-        uncorrelated in estimating the standard error.
+        excess chemical potential. The error estimate assumes that your samples are
+        uncorrelated.
 
         """
-        return self.WidomInsertionPtr.get().measure_excess_chemical_potential(int(reaction_id))
+        if(reaction_id < 0 or reaction_id > (deref(self.WidomInsertionPtr).reactions.size() + 1) / 2):  # make inverse widom scheme (deletion of particles) inaccessible
```

Comment:
I think that the "inverse widom scheme" is not documented anywhere.

Reply:
The inverse widom scheme refers to a scheme like this one: http://www.mtmml.gr/library/downloads/docs/documents/publications/paprer3.pdf
How to use it was documented before but I removed it after talking to Oleg... also since special adaptions to the formulas for calculating the excess chemical potential have to be made in the core for it to be fully correct. Also it is untested. The comments are there, if somebody wanted to reintroduce this scheme. Giving more details here goes beyond this PR.

Reply:
@jonaslandsgesell I thought that there was a consensus that undocumented code and features should not be present in the master.

Reply:
There is no undocumented code here. This is a check for allowed reaction_ids. The comment explains a bit more why the check involves the number 2.